### PR TITLE
Add handwired/bolek keyboard

### PR DESCRIPTION
**Pull request by klausweiss**
## Description

Hi. Adding a support for a handwired keyboard that I've designed. Models for 3D printing the case, as well as wiring schema are available in the keyboard repo: https://github.com/klausweiss/bolek

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/bolek/bolek.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ */
+#define LAYOUT( \
+	K000, K001, K002, K003, K004,                               \
+	K100, K101, K102, K103, K104,                               \
+	K200, K201, K202, K203, K204,                               \
+	            K302, K303, K304,                               \
+	                              K405, K406, K407, K408, K409, \
+	                              K505, K506, K507, K508, K509, \
+	                              K605, K606, K607, K608, K609, \
+	                              K705, K706, K707  \
```
Comment (drashna) [inline keyboards/handwired/bolek/bolek.h:37]: I think you want this.
```suggestion
#define LAYOUT( \
	K000, K001, K002, K003, K004, K405, K406, K407, K408, K409, \
	K100, K101, K102, K103, K104, K505, K506, K507, K508, K509, \
	K200, K201, K202, K203, K204, K605, K606, K607, K608, K609, \
	            K302, K303, K304, K705, K706, K707  \
```

File: keyboards/handwired/bolek/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    /* Base */
+    [_BASE] = LAYOUT(
+        KC_Q, KC_W, KC_E, KC_R, KC_T,
+        KC_A, KC_S, KC_D, KC_F, KC_G,
+        KC_Z, KC_X, KC_C, KC_V, KC_B,
+        KC_ESC, KC_SPC, KC_TAB,
+        KC_Y, KC_U, KC_I, KC_O, KC_P,
+        KC_H, KC_J, KC_K, KC_L, KC_EQL,
+        KC_N, KC_M, KC_COMM, KC_DOT, KC_MINS,
+        KC_ENT, KC_BSPC, RESET)
```
Comment (drashna) [inline keyboards/handwired/bolek/keymaps/default/keymap.c:34]:
```suggestion
    [_BASE] = LAYOUT(
        KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,
        KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_EQL,
        KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_MINS,
                         KC_ESC,  KC_SPC,  KC_TAB,  KC_ENT,  KC_BSPC, RESET
     )
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/bolek/bolek.h (hunk 1 above)
Comment (klausweiss) [reply keyboards/handwired/bolek/bolek.h:37]: Oh, that's perfect. Thanks!

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/bolek/readme.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# bolek
+
+*Low-profile 36% 3d-printed keyboard designed to go with pro micro and kailh chocs*
```
Comment (fauxpark) [inline keyboards/handwired/bolek/readme.md:3]:
```suggestion
Low-profile 36% 3d-printed keyboard designed to go with pro micro and kailh chocs
```

File: keyboards/handwired/bolek/readme.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+* Keyboard Maintainer: [klausweiss](https://github.com/klausweiss)
+* Repository: https://github.com/klausweiss/bolek
+* Hardware Supported: *Pro Micro*
```
Comment (fauxpark) [inline keyboards/handwired/bolek/readme.md:7]:
```suggestion
* Hardware Supported: Pro Micro
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Comment by klausweiss** [comment]
Thank you! (:

**Pull request merged**